content/master/software/install.md

```diff
@@ -244,9 +244,9 @@ at the table below.
 {{< table caption="Feature flags" >}}
```

Comment:
i would expect this page in `master` to be updated with the same content that is in the v1.15 version.  e.g. this block in the 1.15 page about changing the default registry:

* https://deploy-preview-701--crossplane.netlify.app/v1.15/software/install/#change-the-default-package-registry 
* https://github.com/crossplane/docs/pull/701/files#diff-69a7157c116add5db8956596b933da896f47aafaaf82f6f562e6abd1a409e1f2R264-R272

Does not seem to be in this master version of the page, e.g. https://deploy-preview-701--crossplane.netlify.app/master/software/install/#change-the-default-package-registry doesn't exist.

Is that expected for you? and if not, is there possibly more missing content that is in 1.15 but not showing up in master?

Reply:
I'm planning on syncing the `master` folder from `1.15` once all the content is in. right now half the stuff is in master and half in 1.15. 

Because of the way some 1.14 fixes went into master but not 1.14, some in 1.14 and not master and most new content in master, but not the stuff I'm writing it's a bit of a mess. 

Consider the 1.15 directory as the source of truth right now.

[Related discussion](https://github.com/crossplane/docs/issues/600).